doc/src/modules/solvers/solveset.rst

```diff
@@ -285,9 +287,9 @@ What are the general methods employed by solveset to solve an equation?
    recursively.
 
 
- * The function class is now checked if it's trigonometric or hyperbolic, then
-   the function ``_solve_real_trig`` is called, which solves it by converting
-   it to complex exponential form.
+ * The function class is now checked if it's trigonometric or hyperbolic, in
```

Comment:
"The function class is now checked if it's" is verbose. How about here and elsewhere:

```* if the function class is foo then ``bar`` is called to solve it by...```